.travis.yml

```diff
@@ -18,3 +18,5 @@ script:
   - codecov  # Upload the report to codecov.
   - flake8  # Enforce code style ('relaxed' line length limit is set in .flake8 config file).
   - make -C docs html  # Build the documentation.
+  - chmod +x ./nsls2forge_utils/tests/test_check_results.sh  # Add permissions
+  - ./nsls2forge_utils/tests/test_check_results.sh  # Run testing script
```

Comment:
Could this block of shell tests be moved right after the pytest execution (line 16)? That way we won't build the documentation if the test fails.